sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -689,7 +689,6 @@ def __init__(
     self.query_priority = query_priority
     self._job_name = job_name or 'BQ_EXPORT_JOB'
     self._step_name = step_name
-    self._source_uuid = unique_id
```

Comment:
Do we want to remove the input param `unique_id`? Since this is an internal function, I guess it would be okay?

Reply:
yes, we should be able to remove that